README.md

```diff
@@ -23,7 +23,51 @@ juju add-unit -n 2 etcd
 
 Its recommended to run an odd number of machines as it has greater redundancy than even number (ie. 4, you can lose 1 before quorum is lost, where as 5, you can 2).
 
+# The charm told me to see the README
 
+## Usage with your own binaries
+
+This charm supports resources, which means you can supply your own release of
+etcd and etcdctl to this charm. Which by nature makes it highly deployable on
+multiple architectures.
```

Comment:
No. Juju resources are decoupled from package management.  This provides a streamlined way to provide upstream binaries to the charm. It has a fallback mechanism on xenial hosts to reference archive if no resources are supplied.

This wont work on trusty, as there is no etcd package in archive to support the request. This created the install three that has 3 paths.

Upstart + resources
Systemd + resources
Systemd + archive packages

Its not implicitly called out this way, as its preferrable to use the resources that will ship with the charm. As a charm maintainer, we can rev the resources on a development channel of the charm and run smoke tests to verify everything is good, and then upgrade the resource payload on the stable channel of the charm to deliver the latest resource. 

This enables offline deployments as well as supporting upstreams best efforts to have patch releases pushed quickly to consumers of the charm.